criteria.md

```diff
@@ -0,0 +1,47 @@
+### Maturity Level 1 (Sandbox)
+ 
+| Category | Criteria Statement |
+|---|---|
+| Access Control | The project's version control system MUST require multi-factor authentication (MFA) for users when modifying the project repository settings or accessing sensitive data. |
+| Access Control | The project's version control system MUST restrict collaborator permissions to the lowest available privileges by default. |
+| Access Control | The project's version control system MUST prevent accidental direct commits against the primary branch. |
+| Access Control | The project's version control system MUST prevent accidental deletion of the default branch. |
```

Comment:
Are default and primary meant to be different here?

Reply:
unintentional, thanks for catching.

```suggestion
| Access Control | The project's version control system MUST prevent accidental direct commits against the primary branch. |
| Access Control | The project's version control system MUST prevent accidental deletion of the primary branch. |
```